tools/symflower.go

```diff
@@ -91,7 +91,7 @@ func (*symflower) CheckVersion(logger *log.Logger, binaryPath string) (err error
 }
 
 // SymflowerVersionRequired holds the version of Symflower required for this revision of the evaluation.
-const SymflowerVersionRequired = "36800"
+const SymflowerVersionRequired = "37153"
```

Comment:
Would be nice to know what fix... cause otherwise the reason is quite generic... sure we are using a new version because it is better than the old one but that is always the case.

Reply:
The commit message state "The fix makes sure that coverage is still reported even if there are failing tests." Not enough?

Reply:
ah in the message body... sry... yeah...